src/scenes/everest.lua

```diff
@@ -66,7 +68,52 @@ Use the latest ]], { 0.3, 0.8, 0.5, 1 }, "stable", { 1, 1, 1, 1 }, " or ", { 0.8
             clip = false,
             cacheable = false
         }):with(uiu.styleDeep), function()
-            scene.install()
+            -- Check for XNA installed
+            local xnaInstalled =
+                registry.getKey([[HKLM\SOFTWARE\WOW6432Node\Microsoft\XNA\Framework\v4.0\Installed]]) or
+                registry.getKey([[HKLM\SOFTWARE\Microsoft\XNA\Framework\v4.0\Installed]])
+            local install = scene.root:findChild("installs").selected
+            install = install and install.data
+            -- Check for any version before 1.4.0.0
+            local minorVersion = install and tonumber(install.versionCeleste:match("^1%.(%d+)%."))
+            if minorVersion ~= nil and minorVersion < 4 then
```

Comment:
I'm not the hugest fan of that, would prefer being able to pull Everest build metadata (f.e. "required minimum version"). But I don't expect the version number to change any time soon anyway (despite there still being bugs which should be fixed in one more Celeste update), and the current Azure + GitHub combo is quite limiting about that anyway.